{"uuid":"a92e024d-e6cd-47d7-b549-d4fc36620cd5","historyId":"95a6c048d720f16cc2360aaeffebb2d8","fullName":"com.org.testapi.TestForSampleAPIGET.validateUserPostsCommentsExceptionScenarioForNonExistUserId","labels":[{"name":"package","value":"com.org.testapi.TestForSampleAPIGET"},{"name":"testClass","value":"com.org.testapi.TestForSampleAPIGET"},{"name":"testMethod","value":"validateUserPostsCommentsExceptionScenarioForNonExistUserId"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"apc"},{"name":"subSuite","value":"com.org.testapi.TestForSampleAPIGET"},{"name":"host","value":"Sailajas-MacBook-Pro.local"},{"name":"thread","value":"60753@Sailajas-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"To Check Exception Cases","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"class java.util.LinkedHashMap cannot be cast to class java.util.List (java.util.LinkedHashMap and java.util.List are in module java.base of loader 'bootstrap')","trace":"java.lang.ClassCastException: class java.util.LinkedHashMap cannot be cast to class java.util.List (java.util.LinkedHashMap and java.util.List are in module java.base of loader 'bootstrap')\n\tat io.restassured.path.json.JsonPath.getList(JsonPath.java:375)\n\tat com.org.testapi.TestForSampleAPIGET.validateUserPostsCommentsExceptionScenarioForNonExistUserId(TestForSampleAPIGET.java:111)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:566)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1541)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\n\tat org.testng.TestRunner.run(TestRunner.java:596)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\n\tat org.testng.TestNG.run(TestNG.java:1031)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Description : Testing the Post Endpoint whether return proper exception HTTP Status","steps":[],"attachments":[],"parameters":[],"start":1635016111803,"stop":1635016112903}